performance/src/main/java/uk/gov/gchq/palisade/example/perf/PerfApplication.java

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright 2018 Crown Copyright
```

Comment:
```suggestion
 * Copyright 2020 Crown Copyright
```

Reply:
You can hardly tell I copy-pasted the lagacy stuff...